src/fluree/db/ledger/bootstrap.clj

```diff
@@ -222,37 +224,38 @@
-                           (str/split new-db-name #"/"))
-          _             (when (or (txproto/ledger-exists? group network dbid)
-                                  ;; also check for block 1 on disk as a precaution
-                                  (<? (storage/block conn network dbid 1)))
-                          (throw (ex-info (str "Ledger " network "/$" dbid " already exists! Create unsuccessful.")
-                                          {:status 500
-                                           :error  :db/unexpected-error})))
-          master-authid (crypto/account-id-from-message cmd sig)
-          block         (boostrap-memory-db conn [network dbid] {:master-auth-id master-authid :txid txid :cmd cmd :sig sig})
-          new-db        (:db block)
-          block-data    (dissoc block :db)
-          _             (<? (storage/write-block conn network dbid block-data))
-          ;; todo - should create a new command to register new DB that first checks raft
-          _             (<? (txproto/register-genesis-block-async (:group conn) network dbid))
-          ;block-point-success? (async/<! (txproto/propose-new-block-async (:group conn) network dbid block-data))
-          indexed-db    (<? (indexing/index new-db))]
-      ;; write out new index point
-      (<? (txproto/initialized-ledger-async (-> indexed-db :conn :group) txid (:network indexed-db) (:dbid indexed-db)
-                                            (:block indexed-db) (:fork indexed-db) (get-in indexed-db [:stats :indexed])))
-
-      indexed-db)))
-
+   (let [txid          (crypto/sha3-256 cmd)
+         new-db-name   (-> cmd json/parse :db)
+         [network dbid] (if (sequential? new-db-name)
+                          new-db-name
+                          (str/split new-db-name #"/"))
+         _             (when (or (txproto/ledger-exists? group network dbid)
+                                 ;; also check for block 1 on disk as a precaution
+                                 (<? (storage/read-block conn network dbid 1)))
+                         (throw (ex-info (str "Ledger " network "/$" dbid " already exists! Create unsuccessful.")
+                                         {:status 500
+                                          :error  :db/unexpected-error})))
+         master-authid (crypto/account-id-from-message cmd sig)
+         block         (bootstrap-memory-db conn [network dbid] {:master-auth-id master-authid :txid txid :cmd cmd :sig sig})
+         new-db        (:db block)
+         block-data    (dissoc block :db)
+         _             (<? (storage/write-block conn network dbid block-data))
+         ;; todo - should create a new command to register new DB that first checks raft
+         _             (<? (txproto/register-genesis-block-async (:group conn) network dbid))
+                                        ;block-point-success? (async/<! (txproto/propose-new-block-async (:group conn) network dbid block-data))
```

Comment:
Do we need this comment? If so, do you mind changing the semicolon to `;; `, clojure-mode in emacs always tries to indent far to the right comments with a single semicolon.